### PR TITLE
Added in Polkadot's Papers

### LINKED ISSUE #703: Explain differences between original white paper, spec, overview paper, etc.
We should add a page with links to:

1. The original white paper
2. The new overview paper
3. The Polkadot specification

Along with an explanation of what they are and how they differ.

### PR DESCRIPTION
**Pull request by krichard410**
Closes #703 

Changes:
- Added in a new page on Polkadot's papers
- Distinct descriptions of what each paper is and links to those as well

### CODE REVIEW

File: docs/learn-papers.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+id: learn-papers
+title: Polkadot Papers
```
Comment (krichard410) [inline docs/learn-papers.md]: I was unsure of what to actually call this page (the ID and the title), if anyone has any ideas, I would greatly appreciate them!

File: docs/learn-papers.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+is beneficial. This original white paper also explains how each core component in Polkadot works
+together to build this decentralized network. Polkadot's white paper can be found
+[here](https://polkadot.network/PolkaDotPaper.pdf).
+
+## Polkadot's Overview Paper
+
+Polkadot's overview paper is a visual, easy to read, and less technical introduction into its
+blockchain technology. This paper dives into the components of Polkadot but is understandable for
+both a non-technical and technical reader. For reference, the overview paper can be found
+[here](https://polkadot.network/Polkadot-lightpaper.pdf).
```
Comment (lsaether) [inline docs/learn-papers.md]: So actually, the overview paper is not the same as the Light Paper.

The [overview paper](https://github.com/w3f/research/blob/master/docs/papers/OverviewPaper-V1.pdf) is an updated version of the white paper that describes the protocol in more technical terms. We should primarily suggest this paper to people who want to dig more into the protocol.

Comment (krichard410) [reply docs/learn-papers.md]: Oh! Thank you for this, I'll update it. Would it also be helpful to add in the light paper as another section on this page?

Comment (lsaether) [reply docs/learn-papers.md]: Yes, I would suggest to keep the section the same but rename to Polkadot's Light Paper and make a new section for the Overview paper.

File: docs/learn-papers.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Polkadot's original white paper is a technical summary around one possible direction of implementing
+the Polkadot network. This paper uses rationale and technical details to support why this direction
+is beneficial. This original white paper also explains how each core component in Polkadot works
+together to build this decentralized network. Polkadot's white paper can be found
+[here](https://polkadot.network/PolkaDotPaper.pdf).
+
+## Polkadot's Overview Paper
+
+[Polkadot's Overview paper](https://github.com/w3f/research/blob/master/docs/papers/OverviewPaper-V1.pdf)
+is an updated version of the White paper that describes the protocol in more technical terms. We
```
Comment (lsaether) [inline docs/learn-papers.md]:
```suggestion
is an updated version of the white paper that describes the protocol in more technical terms. We
```

File: docs/learn-papers.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Polkadot's light paper is a visual, easy to read, and less technical introduction into its
+blockchain technology. This paper dives into the components of Polkadot but is understandable for
+both a non-technical and technical reader. For reference, the light paper can be found
+[here](https://polkadot.network/Polkadot-lightpaper.pdf).
+
+## The Polkadot Specification
+
+[Polkadot's specification](https://github.com/w3f/polkadot-spec) is a Github Repo that holds the
+latest Polkadot Host Protocol specification, Polkadot's specification tests of the many components
```
Comment (lsaether) [inline docs/learn-papers.md]:
```suggestion
latest Polkadot Host protocol specification, Polkadot's specification tests of the many components
```

**Review by lsaether — APPROVED** [review]
Just needs capitalization normalized - after that LGTM.

File: docs/learn-papers.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+id: learn-papers
+title: Polkadot Papers
+sidebar_label: Polkadot Papers
+---
+
+## Polkadot's White Paper
+
+Polkadot's original white paper is a technical summary around one possible direction of implementing
+the Polkadot network. This paper uses rationale and technical details to support why this direction
+is beneficial. This original white paper also explains how each core component in Polkadot works
```
Comment (Swader) [inline docs/learn-papers.md]:
```suggestion
is beneficial. This original white paper also explains how Polkadot's core components work
```

File: docs/learn-papers.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+together to build this decentralized network. Polkadot's white paper can be found
+[here](https://polkadot.network/PolkaDotPaper.pdf).
```
Comment (Swader) [inline docs/learn-papers.md]: Consider removing this, and linking to it in the beginning of the paragraph like you did with the overview paper. Two reasons - one is consistency, and the other is accessibility. When screen readers go through text, they just read "here" to the people when listing links, unless the link also has a title which in this case it does not. Having the link be something like `[Polkadot's white paper](url)` makes the link more accessible.

File: docs/learn-papers.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+is an updated version of the white paper that describes the protocol in more technical terms. We
+would recommend reading this overview paper if you are interested in digging more into the protocol
+itself.
+
+## Polkadot's Light Paper
+
+Polkadot's light paper is a visual, easy to read, and less technical introduction into its
+blockchain technology. This paper dives into the components of Polkadot but is understandable for
+both a non-technical and technical reader. For reference, the light paper can be found
+[here](https://polkadot.network/Polkadot-lightpaper.pdf).
```
Comment (Swader) [inline docs/learn-papers.md]: Same comment for link as above.

File: docs/learn-papers.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+## The Polkadot Specification
+
+[Polkadot's specification](https://github.com/w3f/polkadot-spec) is a Github Repo that holds the
```
Comment (Swader) [inline docs/learn-papers.md]:
```suggestion
[Polkadot's specification](https://github.com/w3f/polkadot-spec) is a Github repository that holds the
```

File: docs/learn-papers.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+latest Polkadot Host protocol specification, Polkadot's specification tests of the many components
+on the network, and the Polkadot Runtime specification. This repo holds algorithms and explores how
```
Comment (Swader) [inline docs/learn-papers.md]:
```suggestion
of the network, and the Polkadot Runtime specification. This repo holds algorithms and explores how
```

**Review by Swader — CHANGES_REQUESTED** [review]
Minor changes

**Comment by Swader** [comment]
I'm thinking now, do we really need this to be yet another separate page? Perhaps it would be better as part of an existing list of resources, something under the getting started category?

**Comment by krichard410** [comment]
> I'm thinking now, do we really need this to be yet another separate page? Perhaps it would be better as part of an existing list of resources, something under the getting started category?

Ou I second this idea, I think it would be a great addition to the getting started page

**Review by Swader — APPROVED** [review]

**Review by lsaether — APPROVED** [review]

**Pull request merged**